src/App.js

```diff
@@ -43,6 +45,23 @@ export default function App() {
   const initialDate = isValidDate(queryDate) ? queryDate : new Date();
   const [selectedDate, setSelectedDate] = React.useState(initialDate);
 
+  const [totalPages, setTotalPages] = React.useState(0);
+  React.useEffect(() => {
+    (async () => {
+      const url = `${
```

Comment:
n-b This url querying stuff is getting real ugly, something to keep our eye on as we go forward and continue to add new things. into it 😬